.github/issue_template.md

```diff
@@ -0,0 +1,41 @@
+#### A note about questions vs. issues
+
+Please understand that **this is not a forum** but an issue tracker.
+
+The following article explains why you must not ask questions on Github issues: https://medium.com/@methane/why-you-must-not-ask-questions-on-github-issues-51d741d83fde
```

Comment:
You could add a link pointing to the gitter, which is more for questions (am I right @guedou ?)